example-app/SantokuApp/src/apps/app/screens/account/ProfileDetailScreen.tsx

```diff
@@ -3,11 +3,11 @@ import {NativeStackScreenProps} from '@react-navigation/native-stack';
 import {TeamDetailPage} from 'features/team/pages/TeamDetailPage';
 import React, {useCallback} from 'react';
 
-import {RootStackParamList, TeamStackParamList} from '../../navigators/types';
+import {RootStackParamList, AccountStackParamList} from '../../navigators/types';
 
-export const TeamDetailScreen: React.FC<
+export const ProfileDetailScreen: React.FC<
```

Comment:
Should I rename TeamDetailPage to ProfileDetailPage as well?

Reply:
Sorry, I was halfway through the implementation.
499ea9023e630d9323e118f6eef7bc226e701951